test/fuzz/transformation_add_parameter_test.cpp

```diff
@@ -108,53 +107,53 @@ TEST(TransformationAddParameterTest, BasicTest) {
                                                validator_options);
 
   // Can't modify entry point function.
-  ASSERT_FALSE(TransformationAddParameter(4, 60, 12, 61)
+  ASSERT_FALSE(TransformationAddParameter(4, 60, {{0, 7}}, 61)
```

Comment:
I think one of the compilers produces strange errors in release mode if we use maps this way. A solution is to wrap it in an additional pair of braces. `{{{0, 7}}}`.